### PR TITLE
Update feature gates for 1.23

### PR DESCRIPTION
**Pull request by tengqm**
This PR fixes some nits in the feature gates reference page.

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -586,6 +595,12 @@ Each feature gate is designed for enabling/disabling a specific feature:
   (e.g. the cloud-controller-manager) in an HA cluster without downtime.
 - `CPUManager`: Enable container level CPU affinity support, see
   [CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/).
+- `CPUManagerPolicyAlphaOptions`: This allows fine-tuning of CPUManager policies, experimental, Alpha-quality options
+  This feature gate will guard *a group* of CPUManager options whose quality level is alpha.
+  This feature gate will never graduate to beta or stable.
+- `CPUManagerPolicyBetaOptions`: This allows fine-tuning of CPUManager policies, experimental, Beta-quality options
+  This feature gate will guard *a group* of CPUManager options whose quality level is beta.
+  This feature gate will never graduate to stable.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:603]:
```suggestion
- `CPUManagerPolicyAlphaOptions`: This allows fine-tuning of CPUManager policies: experimental, Alpha-quality options.
  This feature gate guards *a group* of CPUManager options whose quality level is alpha.
  This feature gate will never graduate to beta or stable.
- `CPUManagerPolicyBetaOptions`: This allows fine-tuning of CPUManager policies: experimental, Beta-quality options.
  This feature gate guards *a group* of CPUManager options whose quality level is beta.
  This feature gate will never graduate to stable.
```

**Review by sftim — COMMENTED** [review]
/approve

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (bradtopol) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:603]: Hi @sftim looks like you are already engaged on this. Assigning this PR to you.
/assign @sftim

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -889,6 +913,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
   Must be used with `KubeletConfiguration.failSwapOn` set to false.
   For more details, please see [swap memory](/docs/concepts/architecture/nodes/#swap-memory)
 - `NonPreemptingPriority`: Enable `preemptionPolicy` field for PriorityClass and Pod.
+- `OpenAPIEnum`: Enables populating "enum" fields of OpenAPI schemas in the
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Fixed.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Fixed.

**Comment by reylejano** [comment]
Thank you for the fix
/lgtm

**Pull request merged**